docs/UserGuide.md

```diff
@@ -564,6 +575,28 @@ When editing employee and project IDs directly in the data file, take extra caut
 
 [Return to Top](#table-of-contents)
 
+### Traversing command history
+
+HRConnect keeps a log of past commands entered, up to a maximum of the 50 latest commands. To access past commands, make sure that your cursor is in the command box (indicated within the red box below), then press the up (↑) and down (↓) arrow keys. This feature is similar to terminal or command prompt history.
```

Comment:
I think our target audience may not be familiar with terminal or command prompts, would it be better to leave the last sentence out

Reply:
Good point! Updated the user guide.